verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -20,6 +20,22 @@ export const mustBeAuthenticated = (
     res.sendStatus(403);
 };
 
+// mustHaveAnyRole is an express middleware that checks that the currently logged-in user has any of the given roles.
+export const mustHaveAnyRole = (roles: Set<string>) => {
+    return (req: Request, res: Response, next: NextFunction): void => {
+        if (
+            req.isAuthenticated() &&
+            (req.user as UserDocument).roles?.filter((r: string) =>
+                roles.has(r),
```

Comment:
I got confused reading this since roles here has the same name as the user's roles. Could you name it something like requiredRoles to differentiate?

Reply:
Sure, done